### PR TITLE
Nearest Mean Classifier + Dataset collector change to keyboard input

### LINKED ISSUE #44: Rewrite data collector
Rewrite data collector so that one can collect 2 or many classes at the same time
- we could do this through assigning keyboard buttons to each class, or by letting them create an experiment procedure and then running it in a pygame

### PR DESCRIPTION
**Pull request by roenzen1**
Added a nearest mean classifier in classes folder
Not implemented in ViewModel, but worked with personal simple test cases
It's a little wonky to account for edge case of labels being [0 1 5 8] instead of [0 1 2 3 4] 
Should work, should double check

### CODE REVIEW

File: src/Classes/nearestMeanClass.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import numpy as np
+class NearestMean():
```
Comment (zbalkhy) [inline src/Classes/nearestMeanClass.py]: Can we rename this class to MDMClassifier, can we also rename the file to mdmClassifierClass.py

**Comment by roenzen1** [comment]
Also commented out part of kirtana code that was interfering with keypresses and wasn't used (i think it has to do with the matlabplot), doesn't break the program

**Comment by roenzen1** [comment]
It should be good now
